beacon/light/head_tracker.go

```diff
@@ -56,7 +56,7 @@ func (h *HeadTracker) ValidatedHead() (types.SignedHeader, bool) {
 	return h.signedHead, h.hasSignedHead
 }
 
-// ValidatedHead returns the latest validated head.
```

Comment:
The previous method description was correct.

Reply:
Thanks!
Modified.